build.gradle

```diff
@@ -171,6 +171,8 @@ dependencies {
 
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-smile:${jackson_version}"
 
+    implementation group: 'javax.servlet', name: 'javax.servlet-api', version: '3.0.1'
+
```

Comment:
This is not needed since `HttpServletRequest` is not needed in the api; please remove.